src/SonarScanner.MSBuild.Shim/AdditionalFilesService.cs

```diff
@@ -78,11 +78,14 @@ public AdditionalFiles AdditionalFiles(AnalysisConfig analysisConfig, DirectoryI
             return new([], []);
         }
         var allFiles = GetAllFiles(extensions, projectBaseDir);
```

Comment:
Maybe move this right above the `return Partition...`, so that we only do it when we need to.